client/src/pages/Home/Homepage.tsx

```diff
@@ -0,0 +1,22 @@
+import Grid from '@material-ui/core/Grid';
+import useStyles from './useStyles';
+import banner from '../../Images/banner.jpg';
+import { Typography } from '@material-ui/core';
+import FindDogSitter from '../../components/Home/FindDogSitter';
+const Homepage = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Grid container>
+      <Grid item sm={12} md={6} className={classes.findDogSitter} direction="column">
+        <Typography variant="h2">Find the care your dog deserves</Typography>
+        <FindDogSitter />
+      </Grid>
+      <Grid item sm={12} md={6} className={classes.banner}>
+        <img src={banner} className={classes.image} />
```

Comment:
It is a good practice to have 'alt' attributes for HTML image elements

Reply:
Thanks Rajiv, I will make it correct